mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java

```diff
@@ -565,10 +565,18 @@ public void onClick(View view) {
         }
     }
 
+    /**
+     * Shows a fragment that indicates a missing configuration. Displays a hint about no remote
+     * connection when resolveAttempted is set to false.
+     * @param context
+     * @param resolveAttempted Indicate if discovery was attempted, but not successful
+     */
```

Comment:
That Javadoc should go to the indicateMissingConfiguration method (since that's what's called into by the activity) ... just add the `@param` line there.